### DISCUSSION TITLE
Skin editor elements are not shows properly

### DISCUSSION THREAD

**Discussion by mixalisnew** [ppy/osu#24719] [Q&A] [answered]
So in the skin editor when i tried edit a skin which already had an element like the Freedomdive skin a big portion of the element is out of screen. I try to make it fit in the editor screen then on actual gameplay is like half of the screen. ( in case it matters, i play on 16:9 aspect ration and a 1440p monitor)

example=  https://youtu.be/BPBzX54DWJA
(file was to big to upload here)

**Comment by bdach** [comment, marked as answer]
basically duplicate of https://github.com/ppy/osu/discussions/14887

see https://github.com/ppy/osu/discussions/14887#discussioncomment-1514657 for explanation
